### PR TITLE
chore: prepare release 0.15.0

### PR DESCRIPTION
**Pull request by lorenzo-cavazzi**
Prepare release 0.15.0

/deploy #persist

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+* **Renku CLI**: add an option to skip metadata update when executing workflows 🏃
+* **Renku UI**: add support for Mermaid format in Markdown files 🧜‍♀️
+* **Renku UI**: update layout, font, and colors 🎨
+* **Renku UI**: add social links 🔌
+
+Bug fixes
+~~~~~~~~~~
+
+* **User sessions**: prevent logging unnecessary errors when starting new sessions ✏️
+* **Renku CLI**: fix mergetool issues 🧰
```
Comment (rokroskar) [inline CHANGELOG.rst:26]:
```suggestion
* **Renku CLI**: fix merge-tool issues 🧰
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,50 @@
...
+
+This release features improvements and bug fixes to Renku CLI, UI, and User Sessions.
+
+⚠️ Please mind that Renku UI contains changes requiring intervention from an administrator (see below for further details).
+
+Features
+~~~~~~~~~
+
+* **Knowledge Graph**: expose OpenAPI documentation for the lineage endpoint 📃
+* **User sessions**: update base image Dockerfile 🐋
```
Comment (olevski) [inline CHANGELOG.rst:14]:
```suggestion
```

**Review by olevski — CHANGES_REQUESTED** [review]
You can leave that in if you want. 

But it did not trigger a new minor version in the notebooks or Amalthea and I did not call it a feature there. Also not all images were updated. So it is maybe clearer to remove it also because it should not have any effect on end users. The main benefit was less security vulnerabilities.

**Review by olevski — DISMISSED** [review]

**Review by olevski — APPROVED** [review]
🚢

**Review by pameladelgado — APPROVED** [review]
Looks good, great to have the ui charts finally unified 💪 !

**Pull request merged**